packages/ui-react/src/components/Modal/Modal.module.scss

```diff
@@ -8,6 +8,7 @@
   left: 0;
   z-index: variables.$modals-z-index;
   width: 100vw;
+  height: 100vh; // Fallback
```

Comment:
Maybe we should elaborate a little bit on what this serves as a fallback for

Reply:
Christiaan and I talked about it and reasoned to keep it short. If you (and others) don't mind, I will submit this to the OTT repo. Because it is immediately placed before the (original) value, I hope the intention of the comment  is clear enough.

I'll change it of course if somebody else does think it is necessary.